src/OpenTK/Platform/Linux/LinuxDisplayDriver.cs

```diff
@@ -383,11 +344,11 @@ private unsafe static DisplayResolution GetDisplayResolution(ModeInfo* mode)
             return null;
         }
 
-        public override bool TryChangeResolution(DisplayDevice device, DisplayResolution resolution)
+        public bool TryChangeResolution(object device, DisplayResolution resolution)
```

Comment:
This "device" is an internal type for each display driver. There is no shared code, I could make an empty base type to make it a bit more clear though.